__tests__/Employee.test.js

```diff
@@ -0,0 +1,17 @@
+const Employee = require("../lib/Employee");
+
+// describe block for Employee test
+//describe("Employee Class", () => {
+describe("Initialise Employee", () => {
+  // test block (function with two parameters: title, callback )
+  test("should create an object with a name, id and email", () => {
+    // Set up instance of the class with expected result
+    const employee = new Employee("John", 123, "abcdefg@abcdefg.hij");
+
+    // Assertion
+    expect(employee.name).toEqual("John");
```

Comment:
really nice tests Jitka! Great that they test every aspect of the logic inside of your class, fantastic job :-)